src/analysis/retail/evoker/preservation/modules/talents/CallOfYsera.tsx

```diff
@@ -0,0 +1,181 @@
+import { formatNumber, formatThousands } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import { TALENTS_EVOKER } from 'common/TALENTS';
+import { SpellLink } from 'interface';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import { calculateEffectiveHealing } from 'parser/core/EventCalculateLib';
+import Events, { CastEvent, HealEvent } from 'parser/core/Events';
+import DonutChart from 'parser/ui/DonutChart';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import TalentSpellText from 'parser/ui/TalentSpellText';
+import {
+  CALL_OF_YSERA_DREAM_BREATH_INCREASE,
+  CALL_OF_YSERA_DREAM_LIVING_FLAME_INCREASE,
+} from '../../constants';
+import {
+  isFromDreamBreathCallOfYsera,
+  isFromLivingFlameCallOfYsera,
+} from '../../normalizers/CastLinkNormalizer';
+import HotTrackerPrevoker from '../core/HotTrackerPrevoker';
+import DreamBreath from './DreamBreath';
+
+class CallOfYsera extends Analyzer {
+  static dependencies = {
+    dreamBreath: DreamBreath,
+    livingFlame: LivingFlame,
+    hotTracker: HotTrackerPrevoker,
+  };
+  protected hotTracker!: HotTrackerPrevoker;
+  protected dreamBreath!: DreamBreath;
+  protected livingFlame!: LivingFlame;
+  buffedDreamBreaths: number = 0;
+  extraBreathHealing: number = 0;
+  extraBreathHoTHealing: number = 0;
+  buffedLivingFlames: number = 0;
+  extraLivingFlameHealing: number = 0;
+
+  get totalBreathHealing() {
+    return this.extraBreathHealing + this.extraBreathHoTHealing;
+  }
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS_EVOKER.CALL_OF_YSERA_TALENT.id);
+    if (!this.active) {
+      return;
+    }
+    this.addEventListener(
+      Events.cast.by(SELECTED_PLAYER).spell(SPELLS.DREAM_BREATH_CAST),
+      this.onDreamBreathCast,
+    );
+    // this.addEventListener(Events.cast.by(SELECTED_PLAYER).spell(SPELLS.LIVING_FLAME_CAST), this.onLivingFlameCast);
+    //dream breath and dream breath echo healing
+    this.addEventListener(
+      Events.heal.by(SELECTED_PLAYER).spell([SPELLS.DREAM_BREATH, SPELLS.DREAM_BREATH_ECHO]),
+      this.onDreamBreathHeal,
+    );
+    //living flame and living flame echo healing
+    this.addEventListener(
+      Events.heal.by(SELECTED_PLAYER).spell(SPELLS.LIVING_FLAME_HEAL),
+      this.onLivingFlameHeal,
+    );
+    //hot attribution to determine whether a dream breath's hot was buffed by call of ysera or not on application
+
+    //verdant embrace echoes each apply their own call of ysera buff, these are always applied after the initial cast and show up in the log as refresh buff events
+
+    //empowered spell cast ids are applied as removed as a buff event for the duration of their empower
+  }
+
+  onDreamBreathCast(event: CastEvent) {
```

Comment:
This isn't doing much of anything, is it necessary?

Reply:
Ah, you are correct, remnants of previous iteration. Fixed